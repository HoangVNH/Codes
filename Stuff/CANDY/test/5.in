10
6
DVVVVV
6
XVVVVD
2
DD
3
XXV
10
DDXXXVVVDD
10
VVDDDDDVVV
8
DDDVVVVV
5
DDDVV
4
VVVD
8
VVXXXVVV
