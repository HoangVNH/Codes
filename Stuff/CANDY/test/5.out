5
5
0
2
9
9
7
4
3
7
